src/stactools/palsar/cog.py

```diff
@@ -58,7 +64,8 @@ def cogify(tile_path: str, output_directory: str):
             quiet=True,
         )
         logging.info("Wrote out to " + outfile)
-        cogs.append(outfile)
 
-    # return list of cogs by band
+        cogs[band] = outfile
```

Comment:
Returning this as a dict will break the output code. I wasn't sure how to extract only the filenames when it was a dict, hence changing it to a list. 
Please could you update the function code to work with dicts if we are to keep this as a dict?

Reply:
When you say `function` code you mean the Azure Function?

Reply:
I've updated the Az Function, please check.